dlc-manager/tests/ln_dlc_channel_execution_tests.rs

```diff
@@ -259,26 +261,28 @@ impl TestScorer {
     }
 }
 
-impl Score for TestScorer {
+impl ScoreLookUp for TestScorer {
     type ScoreParams = ();
     fn channel_penalty_msat(
         &self,
-        _short_channel_id: u64,
-        _source: &NodeId,
-        _target: &NodeId,
-        _usage: ChannelUsage,
+        _: u64,
+        _: &NodeId,
+        _: &NodeId,
+        _: ChannelUsage,
```

Comment:
not sure why you changed these and not the last one :D (I feel not changing them is also fine)

Reply:
Yeah, that's weird. Definitely not intentional. I'll revert it.